include/picongpu/particles/collision/Collider.def

```diff
@@ -36,22 +36,22 @@ namespace picongpu
              * @tparam T_Species1
              */
             template<typename T_Species0, typename T_Species1>
-            using Pair = boost::mpl::pair<T_Species0, T_Species1>;
+            using Pair = std::pair<T_Species0, T_Species1>;
 
             /* A set of particle filters for a pair of colliding species.
              *
              * @tparam T_Filter0 Filter applied on the 1st species in a pair.
              * @tparam T_Filter1 Filter applied on the 2nd species in a pair.
              */
             template<typename T_Filter0, typename T_Filter1>
-            using FilterPair = boost::mpl::pair<T_Filter0, T_Filter1>;
+            using FilterPair = std::pair<T_Filter0, T_Filter1>;
 
             /* Sets the same particle filter for both species in a pair.
              *
              * @tparam T_Filter A common particle filter for both colliding species.
              */
             template<typename T_Filter>
-            using OneFilter = boost::mpl::pair<T_Filter, T_Filter>;
+            using OneFilter = std::pair<T_Filter, T_Filter>;
```

Comment:
Not sure if this change should be part of this PR. It looks like it should be an independent PR.